Lib/asyncio/streams.py

```diff
@@ -535,6 +535,10 @@ def __init__(self, loop=None, *, _asyncio_internal=False):
             self._loop = events.get_event_loop()
         else:
             self._loop = loop
+            warnings.warn("The loop argument is deprecated since Python 3.8, "
```

Comment:
Please drop this warning.
~FlowControlMixin` is a private API.
Actually, we always pass the loop explicitly in asyncio.

Reply:
ready

Reply:
ready